Automattic/wp-calypso
# No sites message: consolidate layout

Related to https://github.com/Automattic/dotcom-forge/issues/3667#issuecomment-1709574320.

## Proposed Changes

Consolidates the designs between the `/sites` empty state component and the one from `/stats/day`, which is quite old.

| Before | After |
| -------|------|
| <img width="1438" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/c12d0cb1-c744-4895-9dfa-31dbf6a2a6b9"> | <img width="1438" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/20469061-7cb0-4806-a845-1524c52b6d14"> |

## Testing Instructions

Browse `/stats/day` in an account with no sites. You can also verify it shows the new designs in these places with occurrences:

<img width="416" alt="image" src="https://github.com/Automattic/wp-calypso/assets/26530524/b556fb7b-79fa-4fe2-a0cd-f0d951cfb8a0">
